fix(coral): Fix broken error message handling in some Form elements

programmiri opened the pull request:
# Detailed bug report

https://github.com/aiven/klaw/issues/448

## This change
- updates the `formSchema` objects used in tests. They mostly used `name="name"`, this covered a bug in error handling. 
- update `parseFieldErrorMessage` in `Form` to use `errors[name]` (e.g. errors[myFieldName]`) instead of `errors.name`
- remove usage of `const error = get(errors, name)?.message as string;` that now is not needed anymore
- updated `MultiInput` in `Form` to show error messages in some cases: 
    -  error is shown and field marked as invalid when user wants to submit an empty MultiInput that is required 
    - error is shown when user adds something to MultiInput that may not be valid and tabs out
    - error is shown (for an required MI) when user adds something and removes it again 
    

## Screenshots for this implementation

<img width="547" alt="Screenshot 2023-01-19 at 15 56 19" src="https://user-images.githubusercontent.com/943800/213482190-37325d98-d823-4abe-a2d1-25ea61ac0ebd.png">


Resolves: #448

mathieu-anderson reviewed (commented):
It looks good! However, I found myself having to add this code to `MultiInput` to render errors properly in the ACL request form PR: https://github.com/aiven/klaw/pull/398/files#diff-95198cce350bc890a6144a87b5502b4f59d0e6ec618a61999abf5789e6b0b77bR257-R309

Is this unnecessary overengineering, or should it be added to this PR, as it is tightly related?

mathieu-anderson commented:
@programmiri unfortunately not in a separate commit, sorry :/

programmiri commented:
@mathieu-anderson, not a problem, I copied it! 
Should the validation work onBlur? 🤔  For me it only works when I add something something and remove it again 🤔

mathieu-anderson commented:
@programmiri When you say `onBlur`, you mean in the flow like this?
- focus field
- unfocus field
- should trigger "cannot be empty" error

I think this doesn't work indeed. It only works `onChange`, I believe, so you need to have entered things in the input before it starts validating. You can see the behaviour in the ideo posted on the ACL request form PR https://github.com/aiven/klaw/pull/398

programmiri commented:
@mathieu-anderson  - we answered in parallel :D I just updated my post above yours. 
I updated the PR, maybe you can do an 👀 again and if it's ok 👍  it and 🚢  it! (well, merge it, but there's no emoji for that)

mathieu-anderson reviewed (approved):
Nice ✨ Thank you!
